contracts/erc721/InvoiceUnpaidNFT.sol

```diff
@@ -127,7 +127,7 @@ contract InvoiceUnpaidNFT is Initializable, UserMintableERC721 {
 
     UserMintableERC721.initialize(
       "Centrifuge Unpaid Invoices",
-      "CENT_INVOICE_UNPAID",
+      "CNT_INV_UNPD",
```

Comment:
I feel may be we can keep `CENT` and make the name something like `CENT_UPI` (Centrifuge UnPaid Invoice)